packages/ui/tailwind-utils-config/components/link.ts

```diff
@@ -0,0 +1,56 @@
+export default {
+  '.cn-link': {
+    display: 'inline-flex',
+    alignItems: 'center',
+    gap: 'var(--cn-link-gap-default)',
+    width: 'fit-content',
+    font: 'var(--cn-comp-link-default)',
```

Comment:
can we add the font in `packages/ui/tailwind-utils-config/utilities/typography.ts` and use it like 

```js
{
 // ...
'@apply font-link-default': ''
}

```

Reply:
@praneshg239 to be honest, I don't get what the benefit of the `typography.ts` 🤷‍♂️ These two lines are equal:
```css
 font: 'var(--cn-comp-link-default)',
 ***
 '@apply font-link-default': ''
 ````
 Why is one better than another?

Reply:
Fixed